auditbeat/auditbeat.reference.yml

```diff
@@ -1222,6 +1222,17 @@ logging.files:
 # Set to true to enable the monitoring reporter.
 #monitoring.enabled: false
 
+# If output.elasticsearch is enabled, monitoring data for this Auditbeat instance
```

Comment:
Did you mean to include both statements? I think there is too much redundancy, and I'm still not happy with the wording (even the bits that I've contributed). I know you're eager to get this merged. I'd suggest removing the redundant wording (maybe even go with your original statement), and I can improve the wording when I add more info to the docs.

Reply:
I did not, this might be the result of a bad rebase. Fixing...

Reply:
@dedemorton I've removed the duplicate comment in 1e2585f. Thanks for catching this!